datascience/util.py

```diff
@@ -53,3 +54,47 @@ def plot_cdf_area(rbound, lbound=None, mean=0, sd=1):
     plt.title("Normal Curve ~ ($\mu$ = {0}, $\sigma$ = {1}) "
               "{2} < z < {3}".format(mean, sd, llabel, rbound), fontsize=16)
     plt.show()
+
+
+def table_apply(table, func, subset=None):
+    """Applies a function to each column and returns a Table.
+
+    Uses pandas `apply` under the hood, then converts back to a Table
+
+    Parameters
+    ----------
+    table : instance of Table
+        The table to apply your function to
+    func : function
+        Any function that will work with DataFrame.apply
+    subset : list | None
+        A list of columns to apply the function to. If None, function
+        will be applied to all columns in table
+
+    Returns
+    -------
+    tab : instance of Table
+        A table with the given function applied. It will either be the
+        shape == shape(table), or shape (1, table.shape[1])
+    """
+    import pandas as pd
+    from datascience import Table
```

Comment:
Yeah, that's the basic idea. We aren't doing it yet because it was just discussed. It's not super high on the priority list at the moment but I expect it to happen in a few weeks as we set up versioning.